### PR TITLE
feat: Initialize OpenVEX feed

### PR DESCRIPTION
**Pull request by shafeeqes**
**What this PR does / why we need it**:
Initialize OpenVEX feed.

#SecuritySlam

**Which issue(s) this PR fixes** :
Fixes part of #2274

### CODE REVIEW

**Comment by mrueg** [comment]
/lgtm 

Thanks for your contribution!

File: .openvex/templates/main.openvex.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "@context": "https://openvex.dev/ns/v0.2.0",
+  "@id": "https://openvex.dev/docs/public/vex-2912204db7d51d98234a931d600f7cc2dd0bf24a5b5b326de138d64b30c22911",
+  "author": "vexctl (automated template)",
```
Comment (rexagod) [inline .openvex/templates/main.openvex.json:4]: Not sure if this was intentional?

**Review by rexagod — COMMENTED** [review]
Hello, @shafeeqes. Thank you for your work here!

I'm not quite sure though, what the `json` template is for? IIUC `openvexctl` can create manifests for us which can be stored in `./.openvex` and attested to KSM images (however, since KSM images are generated from [`k8s.io`](https://github.com/rexagod/k8s.io/blob/fac06943925f43d0435dbd2435b9c3834fc05e20/registry.k8s.io/images/k8s-staging-kube-state-metrics/images.yaml), do we know if that respects (picks up) the manifests from here)?

**Pull request merged**